style.css

```diff
@@ -0,0 +1,39 @@
+.first-section {
+  border: 4px solid red;
+  font-size: 1.2rem;
+  margin-bottom: 20px;
+  text-align: center;
+}
+.link {
+  color: rgb(197, 62, 186);
```

Comment:
You have to make changes in the whole file and change the color to its hex value.